From 80efb57cace38e4002b4020a786f988d9e03630d Mon Sep 17 00:00:00 2001
From: Dirk Mueller <dirk@dmllr.de>
Date: Fri, 30 Oct 2020 15:46:27 +0100
Subject: [PATCH] Add suse sles 15.2 to the EOL list as well

without that you get this arning:
  WARN	This OS version is not on the EOL list: suse linux enterprise server 15.2

which is actually misleading because 15.2 is the most current release,
we just don't know when it ends. we can however assume that it runs
for at least another year.

Signed-off-by: Dirk Mueller <dirk@dmllr.de>
Signed-off-by: Dirk Mueller <dmueller@suse.com>
---
 pkg/detector/ospkg/suse/suse.go | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/pkg/detector/ospkg/suse/suse.go b/pkg/detector/ospkg/suse/suse.go
index 114eb40..6216840 100644
--- a/pkg/detector/ospkg/suse/suse.go
+++ b/pkg/detector/ospkg/suse/suse.go
@@ -39,7 +39,9 @@ var (
 		"15":   time.Date(2019, 12, 31, 23, 59, 59, 0, time.UTC),
 		"15.1": time.Date(2021, 1, 31, 23, 59, 59, 0, time.UTC),
 		// 6 months after SLES 15 SP3 release
-		// "15.2":   time.Date(2028, 7, 31, 23, 59, 59, 0, time.UTC),
+		"15.2": time.Date(2021, 10, 31, 23, 59, 59, 0, time.UTC),
+		// 6 months after SLES 15 SP4 release
+		// "15.3":   time.Date(2028, 7, 31, 23, 59, 59, 0, time.UTC),
 	}
 
 	opensuseEolDates = map[string]time.Time{
-- 
2.29.0

